src/material-experimental/mdc-menu/_menu-theme.scss

```diff
@@ -44,7 +44,7 @@
   @include mat-using-mdc-typography($config) {
     @include mdc-menu-surface-core-styles($mat-typography-styles-query);
 
-    .mat-mdc-menu-content {
+    .mat-mdc-menu-content, .mat-mdc-menu-item {
```

Comment:
I think the assumption here was that it would be inherited, but I guess that's not the case,